.env

```diff
@@ -8,4 +8,6 @@
 # reference these attributes in code like this:
 # config.fog_directory  = ENV['S3_BUCKET']
 
-S3_BUCKET=bogus-key-for-example
\ No newline at end of file
+SECRET_KEY_BASE=super-fake-bogus-secret-key-base
```

Comment:
The changes to `SECRET_KEY_BASE` and `S3_BUCKET` seem unrelated to this PR.

Reply:
So the issue with this/why we included this fake secret key here is that when "SECRET_KEY_BASE" was accessed with the "fetch" method rather than through normal hash access in ```config/secrets.yml```, it would throw an error unless we actually defined some sort of "SECRET_KEY_BASE" for the fetch to actually grab in the ```.env``` file.

Is this ok to leave in here for now? Or should we just take out the reference to this from ```config/secrets.yml``` in the first place?

Reply:
@JackCurrie this has been addressed in #95. You can rebase on top of the latest `master` and then add the `DATABASE_URL` here and add the `database_url` key _(using `ENV.fetch`)_ to the `defaults` section in `secrets.yml`.